src/index.js

```diff
@@ -0,0 +1,64 @@
+import './styles/index.css';
+import './styles/all.min.css';
+import {
+  addTaskToArray, getTaskFromStore, deleteTaskWith, editTask, clearAllComplete,
+} from './add-remove.js';
+
+const input = document.querySelector('.ph');
+const submit = document.querySelector('.enter');
+const taskEvent = document.body.querySelector('.tasks-list');
+
+// get task from local storage on load

```

Comment:
Please check and remove this comment and other comments althrough your code, use space to separate block of code

Reply:
Hi Victor I think It's good to have comments to guide the reviewer.